### DISCUSSION TITLE
Disable Custom CSS Feature

### DISCUSSION THREAD

**Discussion by morning-knight** [apache/superset#29685] [Q&A / Help]
### Context

As a user of Apache Superset, I would like to enhance the security of my application and prevent users from using custom CSS feature that comes out of the box with superset. 

### Actions taken

- We had a look to see if there's any possible ways to restrict users and completely hide the custom css template feature that allows you to style your dashboards. However without avail.

- We also attempted to restrict it via user permissions on a role, however it doesn't seem to be possible as users can still edit CSS template on their dashboards.

### Question

Is there a way to restrict the usage of custom CSS on dashboards with roles or any feature flag? Otherwise a feature flag within the superset_config removing the ability to use custom css within the application would prove to be beneficial

**Comment by tshallenberger** [comment]
I've created a PR for a feature that will allow users to disable CSS templates (and custom css for dashboards). It's still a WIP, but I'll post here if it's merged!

https://github.com/apache/superset/pull/33675
